packages/integrations/lit/README.md

```diff
@@ -63,33 +63,38 @@ To use your first Lit component in Astro, head to our [UI framework documentatio
 
 However, there's a key difference with Lit _custom elements_ over conventional _components_: you can use the element tag name directly.
 
-Astro needs to know which tag is associated with which component script. We expose this through exporting a `tagName` variable from the component script. It looks like this:
+Writing and importing a Lit component in Astro looks like this:
 
 ```js
 // src/components/my-element.js
 import { LitElement, html } from 'lit';
 
-const tagName = 'my-element';
-
 export class MyElement extends LitElement {
   render() {
     return html` <p>Hello world! From my-element</p> `;
   }
 }
 
-customElements.define(tagName, MyElement);
+customElements.define('my-element', MyElement);
 ```
 
-> Note that exporting the `tagName` is **required** if you want to use the tag name in your templates. Otherwise you can export and use the constructor, like with non custom element frameworks.
-
-In your Astro template import this component as a side-effect and use the element.
+Now, the component can be used via the element tag name
 
 ```astro
+// src/pages/index.astro
 ---
+---
+<!-- note: this way, you can't use client: directives -->
```

Comment:
Could this be a note on the page itself? Explaining why this matters (won't have client-side interactivity)